packages/components/src/components/dropdown/dropdown.scss

```diff
@@ -259,6 +268,10 @@
         min-width: 15rem;
         width: 100%;
 
+        @include mobile {
```

Comment:
Dropdown shouldn't be used in mobile so we can omit @include mobile { ... }

Reply:
this one is the dropdown container ( not native select )